data/formats-data.js

```diff
@@ -980,8 +980,8 @@ exports.BattleFormatsData = {
 		tier: "LC",
 	},
 	cloyster: {
-		randomBattleMoves: ["shellsmash", "hydropump", "rockblast", "iciclespear", "iceshard", "rapidspin", "spikes", "toxicspikes"],
-		randomDoubleBattleMoves: ["shellsmash", "hydropump", "razorshell", "rockblast", "iciclespear", "protect"],
+		randomBattleMoves: ["shellsmash", "liquidation", "rockblast", "iciclespear", "iceshard", "rapidspin", "spikes", "toxicspikes"],
```

Comment:
There's an argument that Hydro Pump lets it hit physically bulky Pokemon, like Skarmory.